### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Tom-A-Watson**

### CODE REVIEW

File: BowlingGame/OpenFrame.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+
+namespace BowlingGame
+{
+
+    // OpenFrame extends the abstract class Frame
```
Comment (KyleAlbiston-TPR) [inline BowlingGame/OpenFrame.cs:6]: avoid committing notes, we can discuss the reason for this next week

File: BowlingGame/SpareFrame.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+
+namespace BowlingGame
+{
+    // SpareFrame extends the abstract class Frame
```
Comment (KyleAlbiston-TPR) [inline BowlingGame/SpareFrame.cs:5]: avoid committing comments

File: BowlingTest/UnitTest1.cs [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        }
+
+        [Test]
+        public void Hookup()
+        {
+            Assert.Pass();
+        }
+
+        [Test]
+        // Ensures balls that end up in the gutter result in a score of 0 (since no pins were hit)
```
Comment (KyleAlbiston-TPR) [inline BowlingTest/UnitTest1.cs]: avoid committing comments

File: BowlingTest/UnitTest1.cs [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        public void GutterBalls()
```
Comment (KyleAlbiston-TPR) [inline BowlingTest/UnitTest1.cs:22]: Test names should be more descriptive. Take a look at your comment above and see how you could turn that into a test name. Do the same with all your tests

**Review by KyleAlbiston-TPR — CHANGES_REQUESTED** [review]
It's a good start, and fundamentally the code looks fine. However, as the game isn't playable yet so I can't review it fully. If you could add in some playability by next Wednesday that would be great.

File: BowlingGame/OpenFrame.cs (hunk 1 above)
Comment (Tom-A-Watson) [reply BowlingGame/OpenFrame.cs:6]: Alright cool, I'm guessing it has something to do with stating obvious things? I know it's bad practice but I only did it to remind myself of the syntax, as I'm unfamiliar with it currently

Comment (KyleAlbiston-TPR) [reply BowlingGame/OpenFrame.cs:6]: Code in a business usually has its own design principles, i.e SOLID, well written code that adheres to your place of works principles shouldn't need any comments. We will look at design principles later on

**Pull request merged**